cronback-lib/default.toml

```diff
@@ -38,3 +38,11 @@ max_triggers_per_tick = 100000
 database_uri = "sqlite://database.sqlite?mode=rwc"
 db_flush_s = 10
 dangerous_fast_forward = false
+
+[project_store]
+cell_id = 0
```

Comment:
I know that I have let this slip before in dispatcher, I don't think `cell_id` makes sense as a configuration key like this. We still need to have a proper design for cluster membership, so I'm peaceful if you don't want to change this. A more realistic design is what you see in `main.dispatcher_cell_map`—from client perspective. 

From server perspective, it's safe to assume—for now—that the project-srv instance will handle all projects until we finalise the design there.

Reply:
I'll keep it here for now given that this exists in all configs as of now.